### PR TITLE
packaging/many: changelog updates for 2.51

### PR DESCRIPTION
**Pull request by anonymouse64**

### CODE REVIEW

**Comment by anonymouse64** [comment]
ah I forgot to change the LP SRU bug number in the changelog, one moment while I force push again

File: packaging/debian-sid/changelog [hunk 1]
```diff
@@ -1,3 +1,234 @@
+snapd (2.51-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - cmd/snap: stacktraces debug endpoint
+    - secboot: deactivate volume again when model checker fails
+    - store: extra log message, a few minor cleanupsAdd a log message
```
Comment (pedronis) [inline packaging/debian-sid/changelog:6]: this entry is mashed up

File: packaging/debian-sid/changelog [hunk 2]
```diff
@@ -1,3 +1,234 @@
...
+      directory
+    - overlord/devicestate: tweak comment about creating recovery
+      systems, formatting tweaks
+    - overlord/devicestate: move devicemgr base suite helpers closer to
+      test suite struct
+    - overlord/devicestate: keep track of tried recovery system
+    - seed/seedwriter: clarify in the diagram when SetInfo is called
+    - overlord/devicestate: add helper for creating recovery systems at
+      runtimeTODO: consider moving this to seedwriter itself with an
+      appropriate name?
```
Comment (pedronis) [inline packaging/debian-sid/changelog:114]: same with the TODO bit

File: packaging/debian-sid/changelog [hunk 3]
```diff
@@ -1,3 +1,234 @@
...
+      cases
+    - o/servicestate/quotas: add functions for getting and setting
+      quotas in state
+    - tests: new buckets for snapd-spread project on gce
+    - spread.yaml: update the gce project to start using snapd-spread
+    - quota: new package for managing resource groups
+    - many: bind and check keys against models when using FDE hooks v2
+    - many: move responsibilities down seboot -> kernel/fde and boot ->
+      secbootThe first two commits are small cleanups, while the latter
+      commits move responsibilities down respectively for:
```
Comment (pedronis) [inline packaging/debian-sid/changelog:147]: same

File: packaging/debian-sid/changelog [hunk 4]
```diff
@@ -1,3 +1,234 @@
...
+    - boot: set extra kernel command line arguments when making a
+      recovery system bootable
+    - boot: pass gadget path to command line helpers, load gadget from
+      seed
+    - tests: new os.paths tool
+    - daemon: make ucrednetGet() return a *ucrednet structure
+    - boot: derive boot variables for kernel command lines
+    - boot,tests: move where we write boot-flags one level up*
+      boot,tests: move where we write boot-flags one level up* cmd/snap-
+      bootstrap/initramfs-mounts: fix boot-flags location from initramfs
```
Comment (pedronis) [inline packaging/debian-sid/changelog:227]: something is a bit weird about this one too but is hard to parse what

**Review by pedronis — COMMENTED** [review]
some entries still seem a bit smashed up

File: packaging/debian-sid/changelog (hunk 4 above)
Comment (anonymouse64) [reply packaging/debian-sid/changelog]: I split this one up/fixed it, some of the things in this were actually also in the 2.50 changelog too

File: packaging/debian-sid/changelog (hunk 3 above)
Comment (anonymouse64) [reply packaging/debian-sid/changelog]: fixed

File: packaging/debian-sid/changelog (hunk 2 above)
Comment (anonymouse64) [reply packaging/debian-sid/changelog]: fixed

File: packaging/debian-sid/changelog (hunk 1 above)
Comment (anonymouse64) [reply packaging/debian-sid/changelog]: fixed

**Review by pedronis — APPROVED** [review]
thanks
